src/main/java/com/jalasoft/selenium/ronald/movies/Customer.java

```diff
@@ -0,0 +1,123 @@
+package com.jalasoft.selenium.ronald.movies;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Customer class.
+ * @author ronald_butron
+ */
+public class Customer {
+    private String name;
+    private List<Rental> rentals;
+    private StringBuilder result;
+
+    /**
+     * Get rental list.
+     *
+     * @return rental list
+     */
+    public List<Rental> getRentals() {
+        return rentals;
+    }
+
+    /**
+     * Constructor.
+     *
+     * @param name customer name.
+     */
+    public Customer(final String name) {
+        this.name = name;
+        this.result = new StringBuilder("Rental Record for " + getName() + "\n");
+        this.rentals = new ArrayList<>();
+    }
+
+    /**
+     * Add rental.
+     * @param arg Rental objected
+     */
+    public void addRental(final Rental arg) {
+        rentals.add(arg);
+    }
+
+    /**
+     * Get costumer name.
+     * @return customer name
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Generate statement value fo account.
+     *
+     * @return statement of account.
+     */
+    public String statement() {
+        double totalAmount = calculateTotalAmount();
+        int frequentRenterPoints = getFrequentPoints();
+
+        return addResultFooterLines(String.valueOf(totalAmount), String.valueOf(frequentRenterPoints));

```

Comment:
Why are you sending as a String?
can we handle as double and int ?
keep in mind the StringBuilder can append double and it as well